ISCSI/SCSI/SCSICommandDescriptorBlock/MMC/GetConfigurationCommand.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using Utilities;
+
+namespace SCSI
+{
+    internal class GetConfigurationCommand : SCSICommandDescriptorBlock
+    {
+        public const int PacketLength = 12;
+        // Request Type
+        public byte RT;
+        // Starting Feature Number

```

Comment:
Let's also implement Starting Feature Number (ushort) and Control (byte) for completeness.